keyboards/atreus/keymaps/ibnuda/config.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+// #include "config_common.h"
```

Comment:
This should be included in the keyboard's `config.h` file; the keymap-level `config.h` file should never even reference it.

```suggestion
```